src/doc/src/commands/cargo-metadata.md

```diff
@@ -83,6 +83,10 @@ The output has the following format:
                        null if not a target dependency.
                     */
                     "target": "cfg(windows)",
+                    /* The file system path for a local path dependency.
+                       not present if not a path dependency.
+                    */
+                    "path": "path/to/dep",
```

Comment:
Should this perhpas be `/path/to/dep` to emphasize that it's always an absolute path?